### PR TITLE
Working on #2972 Template Conversion to Twig Format (usercp_options)

### PR DESCRIPTION
**Pull request by Shade-**
For #2972. Templates converted:
- [x] usercp_options
- [x] usercp_options_date_format
- [x] usercp_options_language
- [x] usercp_options_language_option
- [x] usercp_options_pms
- [x] usercp_options_pms_from_buddys
- [x] usercp_options_pppselect
- [x] usercp_options_pppselect_option
- [x] usercp_options_quick_reply
- [x] usercp_options_style
- [x] usercp_options_timezone
- [x] usercp_options_timezone_option
- [x] usercp_options_time_format
- [x] usercp_options_tppselect
- [x] usercp_options_tppselect_option
- [x] usercp_themeselector
- [x] usercp_themeselector_option

### CODE REVIEW

**Comment by euantorano** [comment]
Regarding the `$lang->sprintf` comment @Shade-, the syntax is:

```twig
{{ trans('lang_variable_name', param1, param2, param3) }}
```

This is the same as:

```php
$lang->sprintf($lang->lang_variable_name, $param1, $param2, $param3);
```

**Comment by Shade-** [comment]
Yep, as you mentioned in Discord, I have already updated this PR accordingly :)

**Comment by euantorano** [comment]
Yep, we should try and document this kind of thing. Perhaps we should add some Wiki pages or doc pages which explain our custom Twig extensions and the functions/filters they provide.

File: inc/views/base/usercp/options.twig [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+                                        </tr>
+                                        {% if stylelist %}
+                                            <tr>
+                                                <td colspan="2"><span class="smalltext">{{ lang.style }}</span></td>
+                                            </tr>
+                                            <tr>
+                                                <td colspan="2">{{ stylelist|raw }}</td>
+                                            </tr>
+                                        {% endif %}
+                                        {% if languages|length > 1 %}
```
Comment (euantorano) [inline inc/views/base/usercp/options.twig]: Alternative syntax:

```twig
{% if languages is not empty %}
```

File: inc/views/base/usercp/options.twig [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+                                                </td>
+                                            </tr>
+                                        {% endif %}
+                                    </table>
+                                </fieldset>
+                            </td>
+                        </tr>
+                    </table>
+
+					<br />
```
Comment (euantorano) [inline inc/views/base/usercp/options.twig]: Indenting here has gone haywire too.

File: usercp.php [hunk 3]
```diff
@@ -446,254 +446,23 @@
...
-            }
-        }
-        eval("\$pppselect = \"".$templates->get("usercp_options_pppselect")."\";");
-    }
-
     $plugins->run_hooks('usercp_options_end');
 
-    eval("\$editprofile = \"".$templates->get("usercp_options")."\";");
-    output_page($editprofile);
+    output_page(\MyBB\template('usercp/options.twig', [
```
Comment (euantorano) [inline usercp.php]: Feels nice to remove this much code 😆

Comment (Shade-) [reply usercp.php]: Indeed 🎉

**Comment by Shade-** [comment]
Done.

**Pull request merged**